pages/translations/ar/LC_MESSAGES/messages.po

```diff
@@ -91,12 +91,12 @@ msgid "Animations and interactions"
 msgstr "الرسوم المتحركة والتفاعلات"
 
 #: /content/amp-dev/about/stories.html:255
-msgid "Check out AMP stories on the web"
-msgstr "مراجعة Web Stories على الويب"
+msgid "Check out Web Stories on the web"
+msgstr "راجع قصص الويب على الإنترنت"
```

Comment:
@ilyaspiridonov I did not believe "Web Stories" should be translated, as it is a proper noun.  Is this correct?

Reply:
that's correct

Reply:
@sebastianbenz @patrickkettner Thanks, I'll update this shortly! Sebastian, our German team are saying that perhaps in German it would make sense to use Web Storys, but please let me know if you think the English version would work better (I saw that it was replaced with Web Stories across all German texts).

Reply:
@patrickkettner updated Web Stories!